fecgov/openFEC
# Add ofec_candidate_totals_detail_mv

## Summary (required)
Candidate financial data is not correct on candidate profile financial total summary. To fix this,
we create this new **ofec_candidate_totals_detail_mv** to replace existing **ofec_totals_candidate_committees_mv** 

ofec_candidate_totals_detail_mv is based on these MVs:
ofec_cand_cmte_linkage_mv
ofec_totals_combined_mv
ofec_agg_coverage_date_vw

- Resolves [#[_3890_]](https://github.com/fecgov/openFEC/issues/3890)


## How to test the changes locally
check new ofec_candidate_totals_detail_mv
1)make sure finance total data correct for candidate by candidate 
both on 
two_election_year(election_full=false)
candidate_election_year (election_full=true).

2)check for 'P', 'H', 'S' and special election(odd number or even number) candidates.

3)check for Puerto Rico candidate

ex: P00003392
S0NY00188
H8GA06195
C00674580
H6PR00082 (Puerto Rico)
